sources/HUBViewModelDiff.m

```diff
@@ -37,6 +38,12 @@
     return [indexPaths copy];
 }
 
+@interface  HUBViewModelDiff ()
+
+@property (nonatomic, assign) BOOL headerComponentIdentifierHasChanged;
```

Comment:
I think it could be useful to expose it for the sake of clarity, but I'm fine either way. I'd rather it be `readonly` though 😛